### PR TITLE
[skip-ci][PyROOT] Release notes for 6.22

### PR DESCRIPTION
**Pull request by etejedor**

### CODE REVIEW

File: README/ReleaseNotes/v622/index.md [hunk 1]
```diff
@@ -287,6 +287,31 @@ Even more includes will be "hidden" when ROOT configured with `-Ddev=ON` build o
 In that case ROOT uses `#ifdef R__LESS_INCLUDES` to replace unused includes by class forward declarations.
 Such `dev` builds can be used to verify that ROOT-based code really includes all necessary ROOT headers.
 
+### Multi-Python PyROOT
+
+In 6.22, the new (experimental) PyROOT is built by default. In order to build with the old PyROOT instead, the option
+`-Dpyroot_legacy=ON` can be used.
```
Comment (oshadura) [inline README/ReleaseNotes/v622/index.md:293]: Do we have this option already in 6.22?

Comment (etejedor) [reply README/ReleaseNotes/v622/index.md:293]: Hehe I was expecting this comment :smile: 

No, we don't have the option at the moment, but we will before we release: `pyroot_experimental` needs to disappear and we need to provide another option to build the old PyROOT. The name (`pyroot_legacy`) is a suggestion from my side, but if you have any other proposal let me know.

File: README/ReleaseNotes/v622/index.md [hunk 2]
```diff
@@ -287,6 +287,31 @@ Even more includes will be "hidden" when ROOT configured with `-Ddev=ON` build o
...
+Python3 and Python2 versions that CMake can find. If only one version can be found, PyROOT will be built for only
+that version. Moreover, for a given Python installation to be considered, it must provide both the Python interpreter
+(binary) and the development package.
+
+The user can also choose to build ROOT only for a given Python version, even if multiple installations exist in
+the system. For that purpose, the option `-DPYTHON_EXECUTABLE=/path/to/python_exec` can be used to point to the
+desired Python installation.
+
+If the user wants to build PyROOT for both Python2 and Python3, but not necessarily with the highest versions that
+are available on the system, they can provide hints to CMake by using `-DPython2_ROOT_DIR=python2_dir` and/or
```
Comment (maxgalli) [inline README/ReleaseNotes/v622/index.md:306]: should we mention ```PythonX_EXECUTABLE``` as well as a hint to CMake?

Comment (etejedor) [reply README/ReleaseNotes/v622/index.md:306]: We could, yes, could you quickly try it and let me know whether it works?

Comment (etejedor) [reply README/ReleaseNotes/v622/index.md:306]: Or perhaps you know already...

Comment (maxgalli) [reply README/ReleaseNotes/v622/index.md:306]: yes it's what I do every time when I need to point to a specific Python installation

**Pull request merged**